### PR TITLE
Rdm 2302 As a user I want to 'cancel and return' during a create event	

### PR DESCRIPTION
**Pull request by andrewfolga**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-2302

### Change description ###
As a user I want to 'cancel and return' during a create event	

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/app/cases/creator/case-creator-submit.component.ts [hunk 1]
```diff
@@ -74,8 +75,40 @@ export class CaseCreatorSubmitComponent implements OnInit {
...
+            })
+        }, error => {
+          console.log('error=', error);
+          this.alertService.setPreserveAlerts(true);
+          this.alertService.warning(error.message);
+          this.router.navigate(['list/case'])
+        });
+        break;
+      case CaseEditPageComponent.RESUMED_FORM_SAVE:
+      this.saveDraft().call(null, event.data).subscribe(_ => {
```
Comment (vlaurin) [inline src/app/cases/creator/case-creator-submit.component.ts]: Missing indentation

File: src/app/shared/case-editor/case-edit.component.ts [hunk 2]
```diff
@@ -88,8 +94,12 @@ export class CaseEditComponent implements OnInit {
     this.fieldsPurger.clearHiddenFields(this.form, this.wizard, this.eventTrigger, currentPageId);
     this.registrarService.reset();
 
+    console.log('router.url=', this.router.url);
+    console.log('navigationOrigin=', this.navigationOrigin);
```
Comment (vlaurin) [inline src/app/shared/case-editor/case-edit.component.ts]: Please remove the console logs.

File: src/app/shared/case-editor/case-edit.component.ts [hunk 3]
```diff
@@ -101,14 +111,16 @@ export class CaseEditComponent implements OnInit {
...
+    theQueryParams['ORIGIN'] = this.navigationOrigin;
+    return this.router.navigate([previousPage.id], { queryParams: theQueryParams, relativeTo: this.route });
   }
 
   hasPrevious(currentPageId: string): boolean {
     return this.wizard.hasPreviousPage(currentPageId, this.fieldsUtils.buildCanShowPredicate(this.eventTrigger, this.form));
   }
 
-  cancel(): void {
+  cancel(): void  {
```
Comment (vlaurin) [inline src/app/shared/case-editor/case-edit.component.ts]: why the extra space?

File: src/app/shared/case-editor/case-edit.component.ts (hunk 2 above)
Comment (andrewfolga) [reply src/app/shared/case-editor/case-edit.component.ts]: Ups. Done now.

File: src/app/shared/case-editor/case-edit.component.ts (hunk 3 above)
Comment (andrewfolga) [reply src/app/shared/case-editor/case-edit.component.ts]: Done now.

**Comment by andrewfolga** [comment]
As discussed we will keep the ORIGIN query parameter but just lowercase it and also do the same for DRAFT query parameter.

**Pull request merged**